### ISSUE TITLE
use cfn-lint

### ISSUE THREAD

**Issue by andreineculau** [tobiipro/support-firecloud#93]
see https://aws.amazon.com/blogs/mt/git-pre-commit-validation-of-aws-cloudformation-templates-with-cfn-lint/

**Comment by IanSavchenko** [comment]
Would be cool!

**Comment by andreineculau** [comment]
I had a look at the codebase and also at the reported issues https://github.com/aws-cloudformation/cfn-python-lint/issues?q=is%3Aissue+is%3Aclosed .

Unfortunately I have to close this as wontfix. I will not put in the effort for this because I think it will simply annoy people: false positives, bugs, get stuck, etc

Just as a knowledge-note: I believe the first step in the right direction would be for AWS to spit out official JSON Schemas. That could take care of 80% of the checking. Assuming that they don't use some sort of schema on the cloudformation backend, they could also start using JSON Schema there. Once that is done, only then introduce business-logic checks in a validation tool to mimic the requirements and/or backend's realities. As it is right now, https://github.com/aws-cloudformation/cfn-python-lint is more of a poor man's linter with an official stamp. https://github.com/fungusakafungus/cloudformation-jsonschema is a thousand times more nobel effort.
